test/e2e_node/runner/run_e2e.go

```diff
@@ -96,11 +92,26 @@ func main() {
 		WaitForUser()
 	}
 
-	// Wait for all tests to complete and check for failures
+	// Wait for all tests to complete and emit the results
 	errCount := 0
 	for i := 0; i < len(hs); i++ {
-		if <-results != nil {
+		tr := <-results
+		host := tr.fullhost
+		if tr.err != nil {
 			errCount++
+			glog.Infof("%s================================================================%s", blue, noColour)
+			glog.Infof("Failure Finished Host %s Test Suite %s %v", host, tr.testCombinedOutput, tr.err)
+			glog.V(2).Infof("----------------------------------------------------------------")
+			glog.V(5).Infof("Host %s Etcd Logs\n%s%s%s", host, gray, tr.etcdCominedOutput, noColour)
+			glog.V(5).Infof("----------------------------------------------------------------")
+			glog.V(5).Infof("Host %s Apiserver Logs\n%s%s%s", host, gray, tr.apiServerCominedOutput, noColour)
+			glog.V(5).Infof("----------------------------------------------------------------")
+			glog.V(2).Infof("Host %s Kubelet Logs\n%s%s%s", host, gray, tr.kubeletCominedOutput, noColour)
```

Comment:
Could we upload it to google storage as other jenkins builds? 
 1. Our logs tend to be very chatty for the purpose of debugging. 
 2. We remove the google storage buckets less often, so the data is more durable.